data/serializer/src/main/java/tech/pegasys/teku/api/response/v1/beacon/GetStateValidatorResponse.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright 2020 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+
+package tech.pegasys.teku.api.response.v1.beacon;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class GetStateValidatorResponse {
```

Comment:
Just curious: I saw the same pattern in the other PR as well. Why do we have a ValidatorResponse class that gets wrapped up in a GetStateValidatorResponse class without adding any other variables or methods?

Reply:
everything is wrapped by a `data`, I'm not sure if i can do it another way, but this is the only way I've found so far...